tox.ini

```diff
@@ -4,38 +4,48 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27, py34, py35, py36, py37
+envlist = py27, py35, py36, py37, {py35, py36, py37}-async
 
 [testenv]
 commands = coverage erase
-           coverage run -m unittest discover -v []
-           coverage report
+           coverage run -m unittest discover -p "test_*.py" -v []
+           coverage report --ignore-errors
+async_commands = coverage run -m unittest discover -p "async_test_*.py" -v []
 deps = -rrequirements.txt
        coverage
-
+async_deps = -rrequirements.txt
+              coverage
+              asynctest
+              python-http-client[async]==3.2.6dev5
```

Comment:
Better be covered by `extras` setting in async envs.

Reply:
`extras` requires adding the test dependencies to the `setup.py`. It doesn't match the maintainer's approach. Actually the entire `tox.ini` can be replaced with the following lines to avoid lots of env sections and optional params.
```
[tox]
envlist = py27, py35, py36, py37, py38, {py35, py36, py37, py38}-async

[testenv]
commands = coverage erase
           !async: coverage run -m unittest discover -p "test_*.py" []
           async:  coverage run -m unittest discover -p "*test_*.py" []
           !py27:  coverage report
           py27:   coverage report --ignore-errors

deps = -rrequirements.txt
       coverage
       async: asynctest
       async: python-http-client[async]==3.2.6dev5
       py27: mock
```